ts/__tests__/Iterable.spec.ts

```diff
@@ -239,6 +239,7 @@ test('iterableConfig_noParams_defaultValues', () => {
   expect(config.allowedProtocols).toEqual([])
   expect(config.androidSdkUseInMemoryStorageForInApps).toBe(false)
   expect(config.useInMemoryStorageForInApps).toBe(false)
+  expect(config.encryptionEnforced).toBe(false)
 })
 
 test('iterableConfig_noParams_defaultDictValues', () => {
```

Comment:
Missing the encryptionEnforced in the next test.

Reply:
Nice catch 👍🏼